crates/superposition_types/src/lib.rs

```diff
@@ -243,9 +244,59 @@ impl Display for RegexEnum {
     }
 }
 
-#[derive(Clone, Deserialize)]
+#[derive(Clone, Debug, Deserialize)]
+pub enum HeadersEnum {
+    ConfigVersion,
```

Comment:
since, we are adding this, can we also replace all the usages of "x-config-version" with this Enum